app/gui/src/model/module/synchronized.rs

```diff
@@ -426,36 +421,37 @@ impl Module {
                     };
                     //id_map goes first, because code change may alter its position.
                     let edits = vec![id_map_change, code_change];
-                    let summary = &summary.summary;
-                    let notify_ls = self.notify_language_server(summary, &new_file, edits, true);
+                    let summary_digest = summary.summary.digest;
+                    let notify_ls =
+                        self.notify_language_server(summary_digest, &new_file, edits, true);
                     profiler::await_!(notify_ls, _profiler)
                 }
                 NotificationKind::MetadataChanged => {
                     let edits = vec![TextEdit {
                         range: summary.metadata_engine_range().into(),
                         text:  new_file.metadata_slice().to_string(),
                     }];
-                    let summary = &summary.summary;
-                    let notify_ls = self.notify_language_server(summary, &new_file, edits, false);
+                    let summary_digest = summary.summary.digest;
+                    let notify_ls =
+                        self.notify_language_server(summary_digest, &new_file, edits, false);
                     profiler::await_!(notify_ls, _profiler)
                 }
-                NotificationKind::Reloaded => Ok(ParsedContentSummary::from_source(&new_file)),
+                NotificationKind::Reloaded => {}
```

Comment:
Do we need this kind of notification, then?

Reply:
Yes, it used elsewhere, for example in https://github.com/enso-org/enso/blob/8b4b522fd1279c763c048f109fc57f3b85466412/app/gui/src/presenter/project.rs#L502